### DISCUSSION TITLE
API testing BDD

### DISCUSSION THREAD

**Discussion by thanujamsc** [badeball/cypress-cucumber-preprocessor#1192] [General]
Can I use [cypress-cucumber-preprocessor](https://github.com/badeball/cypress-cucumber-preprocessor) for BDD API testing?
If so, appreciate if you can attach sample code or example.

**Comment by badeball** [comment]
Generally speaking, you can use this preprocessor for anything Cypress is used for. Can you use Cypress for API testing? If yes, then you can use this preprocessor for it as well.
